root/static/scripts/edit/URLCleanup.js

```diff
@@ -1203,6 +1203,17 @@ const CLEANUPS = {
     clean: function (url) {
       return url.replace(/^https?:\/\/([^/]+\.)?genius\.com/, 'https://genius.com');
     },
+    validate: function (url, id) {
+      switch (id) {
+        case LINK_TYPES.lyrics.artist:
+          return /^https:\/\/genius\.com\/artists\/[\w-]+$/.test(url);
+        case LINK_TYPES.lyrics.release_group:
+          return /^https:\/\/genius\.com\/albums\/[\w-]+\/[\w-]+$/.test(url);
+        case LINK_TYPES.lyrics.work:
+          return /^https:\/\/genius\.com\/(?!(?:artists|albums)\/)[\w-]+$/.test(url);
```

Comment:
If I’m correct, work lyrics’ URL should end with `-lyrics`.
```suggestion
          return /^https:\/\/genius\.com\/(?!(?:artists|albums)\/)[\w-]+-lyrics$/.test(url);
```